tests/test_conversationhandler.py

```diff
@@ -1429,3 +1429,35 @@ def test_callback(u, c):
         assert self.current_state[user1.id] == self.STOPPING
         assert handler.conversations.get((0, user1.id)) is None
         assert not self.test_flag
+
+    def test_conversation_handler_run_async_true(self, dp):
+        conv_handler = ConversationHandler(
+            entry_points=self.entry_points,
+            states=self.states,
+            fallbacks=self.fallbacks,
+            run_async=True,
+        )
+
+        all_handlers = conv_handler.entry_points + conv_handler.fallbacks
+        for state_handlers in conv_handler.states.values():
+            all_handlers += state_handlers
+
+        for handler in all_handlers:
+            assert handler.run_async
+
+    def test_conversation_handler_run_async_false(self, dp):
```

Comment:
maybe make the entry_point async, to check that that won't be overriden to `entry_point[0].run_async=False`

Reply:
I've already have this in the test, see line 1450 for setting the `CommandHandler` with `run_async=True` and testing it in lines 1455-1456.